src/docs/software/using/alphafold.md

```diff
@@ -0,0 +1,296 @@
+    DB_PATH=$SCRATCH/af3_db
+    DATA_JSON=/2pv7/2pv7_data.json
+
+    # run alphafold3 singularity container
+    singularity run \
+         --nv \
+         --env JAX_TRACEBACK_FILTERING=off \
+         --bind $SCRATCH/af_input:/root/af_input \
+         --bind $SCRATCH/af_output:/root/af_output \
+         --bind $MODEL_PARAMS_PATH:/root/models \
+         --bind $DB_PATH:/root/public_databases \
+         af3.sif \
+         --norun_data_pipeline \
+         --json_path=/root/af_output/$DATA_JSON \
+         --model_dir=/root/models \
+         --output_dir=/root/af_output
+    ```
+### Best Practices
+
+1. Using `dcp` to maintain your in Databases to `$SCRATCH` or `$GROUP_SCRATCH`
+
+    Stanford Research Computing maintains a copy of the AlphaFold 3 database within
+    the Oak Common Datasets Repository. We recommend, however, intially downloading
+    the AlphaFold 3 database files directly from Google DeepMind to a scratch directory.
+    This reduces the traffic on the network between Oak and Sherlock and is typically
+    faster than  a direct copy. Unmodified files within `$SCRATCH` and `$GROUP_SCRATCH`
+    are purged every 90 days. In order to maintain a complete database you can include
+    the `dcp` command in your pipeline and inference sbatch scripts, which compares
+    your `$SCRATCH` databases with those on Oak and automatically copies over any
+    missing files.
+
+    The `dcp` command is located in the `mpifileutils` module. mpiFileUtils is a
+    suite of MPI-based tools used to manage large datasets. `dcp` functions similar to
+    `cp` but can leverage multiple cpu cores to copy files faster. Here is an example usage for an AlphaFold database located in `$SCRATCH`.
+ 
+   ```
+    module load system mpifileutils
+    srun -n $SLURM_TASKS_PER_NODE dcp --quite /oak/stanford/datasets/common/alphafold3 $SCRATCH/af3_db
+    ```
+
+    Please keep in mind that running `dcp` will increase the runtime of your job 
+    depending on how many files need to be re-copied. You can also run the `dcp` 
+    command separately and periodically from its own sbatch script. 
+
+2. Running on GPUs with CUDA Capability 8.x or higher
+
+   The singularity container has been tested extensively on Sherlock GPU's with 
+   CUDA capability 8.x or higher. These include H100, L40S, RTX 3090, A100, and 
+   A40 model GPU's. New models, such as the H100 and L40S, produce the fastest 
+   runtimes, with older models taking slightly longer. Consumer grade GPU's, such
+   as the RTX 3090, are also sequence limited due to lower GPU memory.
+
+   To run exclusively on a particular GPU model, you can use the SLURM `--constraint`
+   option. This option takes a node's feature as an argument and sets it as a requirement
+   of the job. Use the Sherlock utility `sh_node_feat -h` to see a list of available
+   node features.
+   
+   Multiple job constraints can also be specified with AND (&) and OR (|) operators. For example, 
```

Comment:
Or maybe also use the `GPU_CC` node feature, that directly maps to GPU compute capabilities?
Like
```
#SBATCH --constraint="GPU_CC:8.9|GPU_CC:9.0"
```